src/_sass/components/videoplayer/_introjs.scss

```diff
@@ -78,3 +59,11 @@
     }
   }
 }
+
+.introjs-helperNumberLayer {
+  background: $red;
+  color: $white;
+  border: 0.125rem solid $white;
+  top: -1.25rem;
+  left: -0.25rem;
```

Comment:
Guide items 1-3 on Level 1 actually appear fully on the screen when this style setting is
`left: 0.25rem;`
Otherwise they're still a bit truncated on the left.